benchmark/python/setup.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
I think this package should depend on hail of equal version.

Reply:
the point of this change is to separate benchmark so that we can benchmark any Hail wheel with the latest benchmark, so I want to keep the setup as minimal as possible.

Reply:
I'm strongly opposed to hidden dependencies. I'd be fine with a `hail` requirement of `>=0.2` but I do think we should have the explicit dependency.